### DISCUSSION TITLE
zim not responding and eating cpu

### DISCUSSION THREAD

**Discussion by manfredlotz** [zim-desktop-wiki/zim-desktop-wiki#1780] [General]
zim is not responding and using 100% cpu. I could kill it and start it again.

I like to know what's going on. So, what is the recommended way to debug/diagnose things?

**Comment by jaap-karssenberg** [comment]
Have a look at the debug log what action is happening when this happens.

There is a report of this behavior on windows when running external
applications like version control commands.

Op ma 6 dec. 2021 14:44 schreef manfredlotz ***@***.***>:

**Comment by manfredlotz** [comment]
In my case it is a Redhat 8.4 system and Python is 3.6.8.
I am not sure what you mean with debug log as I hadn't started zim with `-D`. 

After having killed zim I restarted with `-D` so in case it happens again I have some more information.
